index.bs

```diff
@@ -356,8 +356,8 @@ even slightly or slowly, without affecting your driving direction.
 
 The orientation vector of the [=Orientation Sensor=], can be calculated the following way:
 
-The gravity vector points towards the earth's core when mostly stationary and as long as
-we are not at the poles, we have enough vector length to project that gravity vector onto
+The gravity vector points towards the earth's core when mostly stationary and as long as the
+device is not in free fall, there is enough vector length to project that gravity vector onto
```

Comment:
Actually the latter - the thing we want to project - is the magnetic field vector. So we can of course only do that when not at the poles... 

The gravity vector points towards the earth's core when mostly stationary and as long as the device is not in free fall, there is enough vector length to project the magnetic field vector onto the ground plane. 

Note, this will fail at the poles as the magnetic field vector will point mostly in the same direction as the gravity vector and generally be very unreliable.